src/DraxView.tsx

```diff
@@ -392,7 +392,7 @@ export const DraxView = (
 				// Pass the event up to the Drax context.
 				handleGestureEvent(id, event);
 			},
-			33,
+			10,
```

Comment:
This allows dragging to be at least 60 fps. 

i.e. 1000ms/33ms = 30.3fps

Now 100fps max